storage/Action.cc

```diff
@@ -95,11 +99,17 @@ namespace storage
 			sid_t child_sid = actiongraph.get_devicegraph(LHS)->get_impl()[*vi2]->get_sid();
 
 			vector<Actiongraph::Impl::vertex_descriptor> tmp = actiongraph.actions_with_sid(child_sid, ONLY_LAST);
-			if (!tmp.empty())
-			    actiongraph.add_edge(tmp.front(), v);
+			if (!tmp.empty()) {
```

Comment:
Oops. Rubocop made me lazy in checking this stuff. :-) I will fix it in the whole pull request.